dotcom-rendering/src/web/components/ArticleMeta.tsx

```diff
@@ -158,6 +158,15 @@ const metaContainer = (format: ArticleFormat) => {
 	}
 };
 
+const datelineStyles = (palette: Palette, format: ArticleFormat) => css`
+	${format.design === ArticleDesign.LiveBlog &&
```

Comment:
cssOverride was removed from`Dateline` component and now the styling is done in the `Dateline` component. 

But I got your point about moving the condition. Just had a question, in this case if I wanted to do `cssOverrides={format.design === ArticleDesign.LiveBlog && datelineStyles(palette)}` it wouldn't compile because if the result of `format.design === ArticleDesign.LiveBlog` is false the whole statement will be false which is not acceptable for cssOverrides. 
The error is: `Type false | SerializedStyles is not assignable to type SerializedStyles`

How else would you suggest we could do this?

Reply:
I think the issue is perhaps we're explicitly expecting `SerializedStyles`. Maybe if you just don't type these (as per DCR) then the pattern works?

Reply:
In this case I didn't need to do this anymore as cssOverride was removed from Dateline. 
But I just did a test and it seems it'd work if I do it like this (putting it all inside `css()`):
```
cssOverrides={css(format.design === ArticleDesign.LiveBlog && datelineStyles(palette))}
```

Reply:
👍